test/include/session/session_spec.js

```diff
@@ -0,0 +1,160 @@
+const {expect, sinon} = require('../../helpers/spec_helper');
```

Comment:
testing++

Reply:
I wish I could have done more, but I went beyond what I changed so that is an increase :)

Reply:
also moving them out of the world of should and into the land of expect, sinon, mocha, chia, and such.  Not quite to jest but hey